src/app/booking/ActivitiesCalendar.js

```diff
@@ -230,6 +234,107 @@ export function ActivityCard({ activity, targetDate }) {
   );
 }
 
+export function EmblaContainer({ emblaSx: emblaSxOuter, children }) {
+  const theme = useTheme();
+
+  const [viewportRef, embla] = useEmblaCarousel();
+  const { selectedIndex, scrollSnaps, onDotButtonClick } = useDotButton(embla);
+  const [prevBtnEnabled, setPrevBtnEnabled] = useState(false);
+  const [nextBtnEnabled, setNextBtnEnabled] = useState(false);
+
+  const scrollPrev = useCallback(() => {
+    if (embla) embla.scrollPrev();
+  }, [embla]);
+
+  const scrollNext = useCallback(() => {
+    if (embla) embla.scrollNext();
+  }, [embla]);
+
+  const updateButtons = useCallback(() => {
```

Comment:
Is not it simpler to make scroll next always available, but when clicked on last element, make transition to first?